pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +142,36 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+// Plugins include multiple extension points. When specified, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
```

Comment:
+1 to Bobby's suggestion. It may seem redundant, but that is a good thing for user-facing documentation.

(I hope we can distinguish nil slice from empty slice when parsing config.)

Reply:
I think we should change the protocol then. If an extension point is nil or empty, default set of plugins will be used. If no plugins should be invoked at an extension point, a slice that has an empty string (`""`) must be provided.

Reply:
Please ignore my last comment here if you decide to go with a list of "disabled" and "enabled" plugins. With that approach, we do not need to provide a way to say that no plugins should be invoked at an extension point. With that approach, when an extension point that has some default plugins, the plugins must be disabled in order to have no plugins. When the extension has no default plugins, nothing needs to be enabled or disabled in order to have no plugins. So, either an empty or nil slice works.